activity_browser/ui/tables/models/impact_categories.py

```diff
@@ -399,6 +408,30 @@ def modify_cf(self, cf: tuple, uncertainty: dict) -> None:
             data[1] = uncertainty
         signals.edit_method_cf.emit(tuple(data), self.method.name)
 
+    @Slot(tuple, object, name="modifyCfUncertainty")
+    def modify_cf(self, proxy: QModelIndex) -> None:
+        """Update the CF with new data, possibly converting
```

Comment:
I think it would be good to start following a standard format for the method descriptions (something similar to brightway). This doesn't have to be done for every method, but at least for those used, or called externally from the module:
- a brief method description
- description of arguments
- description of errors raised (if raised)
- description of returned object

Reply:
I know it's a pain, but it's good to start somewhere

Reply:
Agree, will update soon. I propose we follow [PEP 257](https://peps.python.org/pep-0257/).
We should also add this to the PR requirements checklist